kuma/wiki/models.py

```diff
@@ -564,7 +564,7 @@ def render(self, cache_control=None, base_url=None, timeout=None):
             # A timeout of 0 should shortcircuit kumascript usage.
             self.rendered_html, self.rendered_errors = self.html, []
         else:
-            self.rendered_html, errors = kumascript.get(self, cache_control,
```

Comment:
Just documenting here what @davidflanagan and I already discussed in an IRC channel. We need to keep the `cache_control` as part of the `kumascript.get` signature and calls.